doc/source/contributing/contributing_prs.md

```diff
@@ -8,7 +8,7 @@ of time and attention.
 
 Contributors are also encouraged to contribute new code to enhance ArviZ's
 functionality, also via pull requests.
-Please consult the [ArviZ documentation](https://arviz-devs.github.io/arviz/)
+Please consult the [ArviZ documentation](https://python.arviz.org/)
```

Comment:
```suggestion
Please consult the {ref}`ArviZ documentation <homepage>`
```

This also needs adding

```
.. _homepage:
```

in line 6 of (leaving a white line between it and the title below) `doc/source/index.rst` to work, can you add that too?